code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -17,10 +17,11 @@
 	var/autoset = TRUE // Whether the door should inherit access from surrounding areas
 
 /obj/item/airlock_electronics/attack_self(mob/user)
-	if(!ishuman(user) || !issilicon(user))
```

Comment:
You need to get rid of this commit with `git rebase`, since this is a symptom of not properly getting your branch up to date

Reply:
oh right. This one came from the master branch for some fucking dumb dumb reason. Thanks git gui. I'll just purge this and do it again properly.